cylc/uiserver/schema.py

```diff
@@ -241,6 +241,48 @@ class Arguments:
     result = GenericScalar()
 
 
+class ValidateReinstall(graphene.Mutation):
+    class Meta:
+        description = sstrip('''
+            Validate, reinstall, then reload or restart as appropriate.
+
+            This command updates a workflow to reflect any new changes made in
+            the workflow source directory since it was installed.
+
+            The workflow will be reinstalled, then either:
+            * Reloaded (if the workflow is running),
+            * or restarted (if it is stopped).
+        ''')
+        resolver = partial(mutator, command='validate_reinstall')
+
+    class Arguments:
```

Comment:
There are more arguments that this shares with `play` - worth extracting out into a class that can be inherited by both `Arguments` classes?